rule.go

```diff
@@ -73,6 +75,12 @@ type Metadata struct {
 	Value string
 }
 
+// Flowbits description.
+type Flowbits struct {
```

Comment:
additions of any new type should also include a Stringer for writing these back out, and be included in the overall Stringer for the Rule struct as well